src/main/kotlin/com/avail/descriptor/methods/A_Definition.kt

```diff
@@ -43,14 +47,42 @@ import com.avail.descriptor.representation.AvailObject
  * @author Mark van Gulik &lt;mark@availlang.org&gt;
  */
 interface A_Definition : A_Sendable {
```

Comment:
Fixed.  Absolutely be picky.  I've fixed maybe a thousand of these in the last few weeks, but unfortunately it's not safe to use a regex to get them all.  And refactoring mangles the formatting pretty badly still, in ways we can't control with the settings dialog (because refactorings use their own hard-coded formatting rules).